### PR TITLE
COMMERCE-6508 Remove product from compare page

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-6508

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp [hunk 1]
```diff
@@ -17,6 +17,8 @@
 <%@ include file="/init.jsp" %>
 
 <%
+CPCompareContentHelper cpCompareContentHelper = (CPCompareContentHelper)request.getAttribute(CPContentWebKeys.CP_COMPARE_CONTENT_HELPER);
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp:20]: I don't get why you need to use this for just a null check

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp (hunk 1 above)
Comment (smottal) [reply modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp:20]: To be sure that we are in the compare page

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by lorenzolapescara** [comment]
Re-sent with automation at #3716

**Comment by lorenzolapescara** [comment]
ci:close

**Pull request closed without merging**